### ISSUE TITLE
How do you install and run this web app?

### ISSUE THREAD

**Issue by E3V3A** [mundanelunacy/covidmaps#17]
Please update your readme with details on:
How do you install and run this web app?

**Comment by mundanelunacy** [comment]
Duplicate of #19

**Comment by E3V3A** [comment]
Not really. This is about showing the installation steps in your readme and telling the user exactly what need to be added and modified etc. API keys? Servers?

**Comment by mundanelunacy** [comment]
fair enough.  Will get on that this weekend.
